src/Http/Response/Format/Format.php

```diff
@@ -47,6 +54,18 @@ public function setResponse($response)
     }
 
     /**
+     * Set the formats' options.
+     *
+     * @param array $options
+     *
+     * @return void
+     */
+    public function setOptions(array $options)
+    {
+        $this->options = $options;
```

Comment:
All others setters are fluent, and they return `$this`.
I suppose the same should happen here